tabular/src/autogluon/tabular/models/fastainn/tabular_nn_fastai.py

```diff
@@ -184,18 +183,12 @@ def _fit(self,
         else:
             self.y_scaler = copy.deepcopy(self.y_scaler)
 
-        if num_cpus is None:
-            num_cpus = defaults.cpus
-        # additional workers are helping only when fork is enabled; in other mp modes, communication overhead reduces performance
-        num_workers = int(num_cpus / 2)
-        if not is_fork_enabled():
-            num_workers = 0
         if num_gpus is not None:
             if num_gpus == 0:
                 # TODO: Does not obviously impact inference speed
```

Comment:
TODO not needed
add TODO to respect `CUDA_VISIBLE_DEVICES` to select proper GPU

Reply:
added